app/src/main/java/com/example/nmc/ProductModel.java

```diff
@@ -21,105 +21,126 @@
     private int PendingOrderItem;
 
 
-    private int  PendingPaymentDuePAmount;
+    private int PendingPaymentDuePAmount;
     private int PendingPaymentDueRAmount;
     private String PendingPaymentVendorP;
     private String PendingPaymentVendorR;
 
-    public void setProductDetails(String ProductName,int ProductSerialNumber)
-    {
-        this.ProductName=ProductName;
-        this.ProductSerialNumber=ProductSerialNumber;
+
+    int i;
+
+    public void setProductDetails(String ProductName, int ProductSerialNumber, String ProductImageURL) {
+        this.ProductName = ProductName;
+        this.ProductSerialNumber = ProductSerialNumber;
+        this.ProductImageURL = ProductImageURL;
     }
 
+
     public void getProductDetails() {
         System.out.println("Product Name=" + ProductName);
         System.out.println("ProductSerialNumber=" + ProductSerialNumber);
+        System.out.println("ProductImageURL" + ProductImageURL);
 
     }
 
 
-    public void  setVendorDetails(String VendorName, int VendorPhoneNumber, String VendorAddress, int vendorGSTNumber )
-    {
-        this.VendorName=VendorName;
-        this.VendorPhoneNumber=VendorPhoneNumber;
-        this.VendorAddress= VendorAddress;
-        this.vendorGSTNumber=vendorGSTNumber;
+    public void setVendorDetails(String VendorName, int VendorPhoneNumber, String VendorAddress, int vendorGSTNumber) {
+        this.VendorName = VendorName;
+        this.VendorPhoneNumber = VendorPhoneNumber;
+        this.VendorAddress = VendorAddress;
+        this.vendorGSTNumber = vendorGSTNumber;
     }
 
-    public void  getVendorDetails()
-    {
-        System.out.println("VendorName=" +VendorName);
-        System.out.println("VendorPhoneNumber="+VendorPhoneNumber);
-        System.out.println("VendorAddress="+ VendorAddress);
-        System.out.println("VendorGSTNumber="+vendorGSTNumber);
+    public void getVendorDetails() {
```

Comment:
Instead of using System.out.println, create a Hashmap, store all the details there like for example if x is the hashmap,
x can be {"Vendor Name": "Abc", "VendorPhoneNumber":12341341.....}, and then return that hashmap. We wont be making any use of printing the details actually. If we are using a getter for any object field, it should return us values instead of printing. Please do the same at all the places wheerever you have used System.out.println